[API Shield] FAQ Page (#5647)
cloudflare/cloudflare-docs: content/api-shield/frequently-asked-questions.md

---
pcx_content_type: FAQ
title: FAQs
weight: 3
layout: list
---
# Frequently Asked Questions

> 

Reviewer: You shouldn't skip to H3s. These should be reformatted as H2s.

2. Requests must return `2XX` response codes from the edge.
3. Requests must not come directly from Cloudflare Workers.
4. At least 500 requests are made to the discovered endpoint within a 10 day period.

### Why do I not receive threshold recommendations for my discovered API endpoints?

Thresholds can only be recommended for endpoints that receive sufficient levels of traffic that meet the following criteria: 

* Only requests with the same criteria as API Discovery are considered. 
* If traffic has been erratic or intermittent to this endpoint, the threshold might not show up. Cloudflare needs endpoints to receive sufficient valid traffic in any 24-hour period in the last 7 days or since the initial discovery of the endpoint to make statistically safe threshold suggestions.
* Cloudflare also requires at least 50 distinct sessions to have accessed the endpoint in any 24-hour period in the last 7 days or since the initial discovery of the endpoint.

If you do not receive threshold recommendations for a discovered endpoint, you will see one of the following error codes: 

> `404 response`: We have not seen sufficient valid traffic for this zone to generate recommendations.

Reviewer: Should format these as a `ul`

> `404 response`: We have not seen sufficient valid traffic for this zone to generate recommendations.

Reviewer: ```suggestion
`404 response`: Cloudflare has not seen sufficient valid traffic for this zone to generate recommendations.
```

> `551 response`: We have successfully generated recommendations at some point in the past, but we have not seen sufficient recent valid traffic to provide up-to-date recommendations.

Reviewer: ```suggestion
`551 response`: Cloudflare has successfully generated recommendations at some point in the past, but we have not seen sufficient recent valid traffic to provide up-to-date recommendations.
```

### Does this work for JDCloud customers?

> Not currently.

Reviewer: Is this the same as China network? If so, should provide a link over to those docs.
